terraform/cluster.tf

```diff
@@ -41,6 +29,26 @@ resource "aws_instance" "app_server" {
   ]
 }
 
+resource "aws_db_instance" "db" {
+  identifier           = "${var.cluster_name}-db-${count.index}"
+  allocated_storage    = 100
+  storage_type         = "io1"
+  iops                 = 1000
```

Comment:
this is hardcoded on purpose I assume?

Reply:
Yeah .. there is some relation with `iops` and `allocated_storage`, if one is 1000, other can't be less than 100. It will cause more confusion if we parameterize this.